.github/workflows/checks.yaml

```diff
@@ -57,7 +57,8 @@ jobs:
         uses: golangci/golangci-lint-action@v3.7.0
         with:
           version: v1.55.2
-          args: --timeout 10m0s
+          args: --timeout=10m0s --go=${{ env.golang-version }}
```

Comment:
is this change needed? If possible it should another PR to have a cleaner change history.

Reply:
Got it.  I will add another PR